manylinux2010-x64/Dockerfile.in

```diff
@@ -4,10 +4,82 @@ MAINTAINER Matt McCormick "matt.mccormick@kitware.com"
 ENV DEFAULT_DOCKCROSS_IMAGE dockcross/manylinux2010-x64
 
 #include "common.manylinux"
+# Image build scripts
```

Comment:
@jcfr: Is this whole section still necessary? It's a fairly old PR and though I understand the `common.label-and-env` part, I am not sure why this part is needed.

If that's needed, then to me it seems ready to merge